packages/components/src/TextField/TextField.test.tsx

```diff
@@ -129,4 +129,52 @@ describe('TextField', () => {
       expect(inputElement.value).toHaveLength(MAX_LENGTH);
     });
   });
+
+  it('При переводе фокуса с инпута пробел в конце текста удаляется', async () => {
+    const onChangeMock = vi.fn();
+
+    renderWithTheme(
+      <TextField
+        inputProps={{ 'data-testid': 'textFieldForTrim' }}
+        onChange={(e) => onChangeMock(e.target.value)}
+      />,
+    );
+
+    const inputElement = screen.getByTestId(
+      'textFieldForTrim',
+    ) as HTMLInputElement;
+
+    await userEvents.type(inputElement, 'test ');
+    await userEvents.tab();
+
+    await waitFor(() => {
+      expect(onChangeMock.mock.lastCall).toEqual(['test']);
+    });
+
+    vi.clearAllMocks();
+  });
+
+  it('При переводе фокуса с инпута вызывается переданный onBlur', async () => {
+    const onBlurMock = vi.fn();
```

Comment:
```suggestion
    const onBlurSpy = vi.fn();
```

https://kaluga-astral.github.io/style-guide/docs/rules/tests/test-double#spy-%D1%88%D0%BF%D0%B8%D0%BE%D0%BD%D1%8B-%D0%B8%D0%BC%D0%B5%D1%8E%D1%82-%D0%BF%D0%BE%D1%81%D1%82%D1%84%D0%B8%D0%BA%D1%81-spy